doped/utils/qehacks.py

```
@@ -0,0 +1,56 @@
+# hacks.py
```

Ruff (0.12.2)

Missing docstring in public module

(D100)